scripts/build_darwin.sh

```diff
@@ -5,9 +5,12 @@ mkdir -p dist
 GO_LDFLAGS="-X github.com/jmorganca/ollama/version.Version=$VERSION"
 GO_LDFLAGS="$GO_LDFLAGS -X github.com/jmorganca/ollama/server.mode=release"
 
+# generate packaged model runner executables
+go generate ./...
+
 # build universal binary
-CGO_ENABLED=1 GOARCH=arm64 go build -ldflags "$GO_LDFLAGS" -o dist/ollama-darwin-arm64
-CGO_ENABLED=1 GOARCH=amd64 go build -ldflags "$GO_LDFLAGS" -o dist/ollama-darwin-amd64
+GOARCH=arm64 go build -ldflags "$GO_LDFLAGS" -o dist/ollama-darwin-arm64
```

Comment:
I think we'll need to cross compile the stuff in `go generate ./...` (or not if it's better to build on the respective arch).

Reply:
This assumes an arm64 build. amd64 will need to be built on an Intel system. This is because cross compiling CMAKE is not as straight forward as Go